content/docs/learning-guide/tutorials/assets/physx-colliders.md

```diff
@@ -2,13 +2,21 @@
 linkTitle: PhysX Assets
 title: Process PhysX Collider Assets
 description: Learn to customize PhysX collider asset processing in Open 3D Engine (O3DE) with Scene Settings.
-weight: 100
+weight: 400
 toc: true
 ---
 
-**Open 3D Engine (O3DE)** has a robust set of options for generating PhysX collider assets. In **Scene Settings**, you can customize PhysX collider asset generation. The generated PhysX colliders are stored in `.pxmesh` product assets. You can add the PhysX collider assets to a **PhysX Mesh Collider** component.
+**Open 3D Engine (O3DE)** has a robust set of options for generating PhysX collider assets. There are three PhysX collider types that you can use in different simulation scenarios. This topic explains the benefits and limitations of the different collider types, as well as the basics of generating PhysX asset colliders with  [**Scene Settings**](/docs/user-guide/assets/scene-settings/scene-settings).
 
-There are three PhysX asset collider types that you can use in different simulation scenarios. This topic explains the benefits and limitations of the different collider types, as well as the basics of generating PhysX asset colliders with Scene Settings.
+The generated PhysX colliders are stored in `.pxmesh` product assets. You can add the PhysX collider assets to a **PhysX Collider** component by selecting **PhysicsAsset** in the component's **Shape** property.

```

Comment:
This incorporates some changes introduced in a [PR](https://github.com/o3de/o3de.org/commit/2d12811ca0cdf54c6d9aa88d5c080cfefed715d6#diff-687a70c2c47e65118bd6661d32ab408798602293203e4400e42055ce77eb83f9R9) to the `development` branch made last week. See the [blame](https://github.com/o3de/o3de.org/blame/development/content/docs/learning-guide/tutorials/assets/physx-colliders.md#L9) for this file's history.
```suggestion
The generated PhysX colliders are stored in `.pxmesh` product assets. You can add the PhysX collider assets to a **PhysX Mesh Collider** component.

```

Reply:
Good catch! But I wonder if the change should be this:

You can add the PhysX collider assets to a **PhysX Mesh Collider** component by selecting **PhysicsAsset** in the component's **Shape** property.

@micronAMZN made the recent change to main and might have some additional insight.